### PR TITLE
4 key request list

### PR DESCRIPTION
**Pull request by Patrick7321**

### CODE REVIEW

**Comment by enki211** [comment]
You did not include keyform/urls or home.html

File: keyform/templates/keyform/home.html [hunk 1]
```diff
@@ -1,6 +1,61 @@
...
+                    <div class="large-2 columns text-center">
+                        {% for key in object.keydata_set.all %}
+                            {{ key.new_core_number }}
+                        {% endfor %}
+                    </div>
+                    <div class="large-1 column text-center">
+                        {{ object.comment_set.all.count }}
+                    </div>
+                    <div class="large-2 columns text-center">
+                        Status
```
Comment (enki211) [inline keyform/templates/keyform/home.html]: This should probably be 
```
{{ object.get_status_display }}
```

File: keyform/templates/keyform/home.html [hunk 2]
```diff
@@ -1,6 +1,61 @@
...
+                    </div>
+                </div>
+            </div>
```
Comment (enki211) [inline keyform/templates/keyform/home.html]: I belive this `</div>` and the `{% endfor %}` should be switched

File: keyform/views.py [hunk 3]
```diff
@@ -2,7 +2,11 @@
 from __future__ import unicode_literals
 
 from django.shortcuts import render
-from django.views.generic.base import TemplateView
+from django.views.generic.list import ListView
+from keyform.models import Request, Comment
```
Comment (enki211) [inline keyform/views.py]: I don't think you need to import the Comment model

File: keyform/views.py [hunk 4]
```diff
@@ -2,7 +2,11 @@
 from __future__ import unicode_literals
 
 from django.shortcuts import render
```
Comment (enki211) [inline keyform/views.py]: You also don't need this

File: keyform/templates/keyform/home.html [hunk 5]
```diff
@@ -1,6 +1,61 @@
...
 {% load i18n %}
+{% load static %}
+{% block css %}
+    {{ block.super }}
+    <link rel='stylesheet' type='text/css' href="{% static 'keyform/css/home.css' %}"/>
+{% endblock %}
 
 {% block content %}
-    {% trans 'Welcome to Keyform!' %}
+    <h2 class="title">Keyform Requests</h2>
```
Comment (enki211) [inline keyform/templates/keyform/home.html]: trans tag

File: keyform/templates/keyform/home.html [hunk 6]
```diff
@@ -1,6 +1,61 @@
...
+                </div>
+                <div class="large-1 columns text-center">
+                    {% trans 'Comments' %}
+                </div>
+                <div class="large-2 columns text-center">
+                    {% trans 'Status' %}
+                </div>
+            </div>
+            <div class="bigobject">
+            {% for object in object_list %}
```
Comment (enki211) [inline keyform/templates/keyform/home.html]: Indentation

File: keyform/templates/keyform/home.html [hunk 7]
```diff
@@ -1,6 +1,59 @@
...
+                    {% for key in object.keydata_set.all %}
+                        {{ key.room_number }}{% if not forloop.last %}, {% endif %}
+                    {% endfor %}
+                </div>
+                <div class="large-2 columns text-center">
+                    {% for key in object.keydata_set.all %}
+                        {{ key.new_core_number }}{% if not forloop.last %}, {% endif %}
+                    {% endfor %}
+                </div>
+                <div class="large-1 column text-center">
```
Comment (enki211) [inline keyform/templates/keyform/home.html]: colums instead of column?

Comment (Patrick7321) [inline keyform/templates/keyform/home.html]: You can use either of them and the definition is the same

File: keyform/static/keyform/css/home.css [hunk 8]
```diff
@@ -0,0 +1,24 @@
+.object-list {
+    padding: 20px;
+}
+
+.header {
+    border-bottom: 1px solid black;
+}
+
+.bigobject {
```
Comment (enki211) [inline keyform/static/keyform/css/home.css]: I don't think this is necessary anymore

**Comment by shaneallgeier** [comment]
The django-debug-toolbar stuff should be part of a seperate PR. In fact, it already is (#9).

File: keyform/templates/keyform/home.html [hunk 9]
```diff
@@ -1,6 +1,59 @@
...
+            <div class="object row">
+                <div class="large-3 columns text-center">
+                    {{ object.created_timestamp|date:"m/d/Y" }}
+                </div>
+                <div class="large-2 columns text-center">
+                    {{ object.building }}
+                </div>
+                <div class="large-2 columns text-center">
+                    {% for key in object.keydata_set.all %}
+                        {{ key.room_number }}{% if not forloop.last %}, {% endif %}
```
Comment (shaneallgeier) [inline keyform/templates/keyform/home.html]: The table is kind of broken if `keydata_set` is empty. You can use the [{% empty %} clause](https://docs.djangoproject.com/en/1.11/ref/templates/builtins/#for-empty) in conjunction with a non-breaking space (`&nbsp;`) to keep this from breaking everything.

File: keyform/templates/keyform/home.html [hunk 10]
```diff
@@ -1,6 +1,59 @@
...
+                    {% endfor %}
+                </div>
+                <div class="large-2 columns text-center">
+                    {% for key in object.keydata_set.all %}
+                        {{ key.new_core_number }}{% if not forloop.last %}, {% endif %}
```
Comment (shaneallgeier) [inline keyform/templates/keyform/home.html]: The table is kind of broken if `keydata_set` is empty. You can use the [{% empty %} clause](https://docs.djangoproject.com/en/1.11/ref/templates/builtins/#for-empty) in conjunction with a non-breaking space (`&nbsp;`) to keep this from breaking everything.

File: keyform/templates/keyform/home.html [hunk 11]
```diff
@@ -1,6 +1,63 @@
...
+    {{ block.super }}
+    <link rel='stylesheet' type='text/css' href="{% static 'keyform/css/home.css' %}"/>
+{% endblock %}
 
 {% block content %}
-    {% trans 'Welcome to Keyform!' %}
+    <h2 class="title">{% trans 'Keyform Requests' %}</h2>
+    <div class="object-list">
+        <div class="header row">
+            <div class="large-3 columns text-center">
```
Comment (tom08) [inline keyform/templates/keyform/home.html]: This display does not scale well to a smaller screen.  The foundation classes do what they're supposed to, but the columns are disassociated from each other in the smaller sizes.

**Pull request merged**